src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1111111,
+    description: "Stub description",
+    icon: "stubIcon",
+    humidity: 30,
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    wind: {
+      speed: 12,
+      direction: "S",
+    },
+    onselect: () => {},
```

Comment:
Since we're stubbing the function, we should use the `jest.fn()` placeholder
```suggestion
    onselect: jest.fn(),
```
Also, this will error as on**s**elect is not defined in the component, but on**S**elect is